docs/diagrams/UiClassDiagram.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+hide class fields
+hide class methods
+
+package Recipe {
+class RecipeList
+class Recipe
+class Ingredient
+}
+
+package Ui {
+class Ui
+class AddMode
+class Editor
+}
+
+class Storage
+
+Ui ..> CommandResult
+
+AddMode --> Recipe
+AddMode --> Ingredient
+RecipeList *-right-> Recipe
```

Comment:
```suggestion
RecipeList -right-> Recipe
```
`RecipeList` contains an attribute that is `ArrayList<Recipe>`

Reply:
Same comment as above that this is a composition. However, thanks for pointing out that it is an `ArrayList` and I should add in the `0..1` multiplicity.